src/compas_eve/core.py

```diff
@@ -1,5 +1,6 @@
 from compas.data import json_dumps
 from compas.data import json_loads
+import msgpack
```

Comment:
Since `msgpack` is not available for ironpython, we should either `try..except` this import or check `if compas.IPY` (and subsequently disable binary serialization if that's the case).